example/spec/parallel/ingestGranule/IngestUMMGSuccessSpec.js

```diff
@@ -0,0 +1,273 @@
+'use strict';
+
+const fs = require('fs-extra');
+const urljoin = require('url-join');
+const got = require('got');
+const cloneDeep = require('lodash.clonedeep');
+const differenceWith = require('lodash.differencewith');
+const isEqual = require('lodash.isequal');
+const {
+  models: {
+    Execution, Collection, Provider
+  }
+} = require('@cumulus/api');
+const {
+  aws: {
+    getS3Object,
+    parseS3Uri
+  },
+  constructCollectionId
+} = require('@cumulus/common');
+const {
+  api: apiTestUtils,
+  buildAndExecuteWorkflow,
+  LambdaStep,
+  conceptExists,
+  getOnlineResources,
+  granulesApi: granulesApiTestUtils
+} = require('@cumulus/integration-tests');
+
+const {
+  loadConfig,
+  uploadTestDataToBucket,
+  deleteFolder,
+  createTimestampedTestId,
+  createTestDataPath,
+  createTestSuffix,
+  templateFile
+} = require('../../helpers/testUtils');
+
+const {
+  setupTestGranuleForIngest,
+  loadFileWithUpdatedGranuleIdPathAndCollection
+} = require('../../helpers/granuleUtils');
+
+const config = loadConfig();
+const lambdaStep = new LambdaStep();
+const workflowName = 'IngestAndPublishGranule';
+
+const granuleRegex = '^MOD09GQ\\.A[\\d]{7}\\.[\\w]{6}\\.006\\.[\\d]{13}$';
+
+const templatedOutputPayloadFilename = templateFile({
+  inputTemplateFilename: './spec/parallel/ingestGranule/IngestGranule.output.payload.template.json',
+  config: config[workflowName].IngestGranuleOutput
+});
+
+const s3data = [
+  '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met',
+  '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf',
+  '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606_ndvi.jpg'
+];
+
+async function getUmmJs(fileLocation) {
```

Comment:
rename `getUmmJson`? 
But I have added a  metadataObjectFromCMRJSONFile(cmrFilename)  to cmrjs 
which is being renamed metadataObjectFromCMRFile(cmrFilename) that will return the object representation of either json or xml cmrfiles.  I think it's fine to leave this here since it's explicit, but if you were interested...

Reply:
It's not getting JSON, though, it's getting javascript.

Reply:
then I clearly didn't understand what it was doing